httpserver/server.go

```diff
@@ -54,25 +54,56 @@ func (s *Server) httpLogger(next http.Handler) http.Handler {
 }
 
 func (s *Server) RunInBackground() {
-	s.log.Info("Starting HTTP server",
-		zap.String("listenAddress", s.cfg.ListenAddr),
-		zap.String("version", s.cfg.Version),
-	)
+	// metrics
+	if len(s.cfg.MetricsAddr) != 0 {
```

Comment:
```suggestion
	ifs.cfg.MetricsAddr != "" {
```

that's a more typical pattern